[2020-06-27 09:26:37,840]{/home/alta5/Desktop/ALTA ONLINE LEARNING/BACK-END/blueprints/__init__.py:86} WARNING - REQUEST_LOG	{"method": "PATCH", "code": "200 OK", "uri": "/mentee/1?", "request": null, "response": {"id": 1, "username": "agusdwis", "full_name": "Agus D Sasongko", "email": "agusdwi@alterra.com", "address": "Solo, Indonesia", "phone": "085635467564", "place_birth": "Karanganyar", "date_birth": "17/08/1995", "avatar": "https://alterra-online-learning.s3-ap-southeast-1.amazonaws.com/avatar/b9010c9fb7ce481eb78a67b32f161f5c_anonymous.png", "background_education": "S1 Teknik Geofisika", "github": "agusdwis", "description": "", "status": true, "created_at": "Sun, 21 Jun 2020 13:22:05 -0000", "update_at": null}}
[2020-06-27 09:28:11,723]{/home/alta5/Desktop/ALTA ONLINE LEARNING/BACK-END/blueprints/__init__.py:86} WARNING - REQUEST_LOG	{"method": "OPTIONS", "code": "200 OK", "uri": "/mentee/1?", "request": null, "response": {}}
[2020-06-27 09:28:11,759]{/home/alta5/Desktop/ALTA ONLINE LEARNING/BACK-END/blueprints/__init__.py:86} WARNING - REQUEST_LOG	{"method": "PATCH", "code": "200 OK", "uri": "/mentee/1?", "request": null, "response": {"id": 1, "username": "agusdwis", "full_name": "Agus D Sasongko", "email": "agusdwi@alterra.com", "address": "Solo, Indonesia", "phone": "085635467564", "place_birth": "Karanganyar", "date_birth": "17/08/1995", "avatar": "https://alterra-online-learning.s3-ap-southeast-1.amazonaws.com/avatar/b9010c9fb7ce481eb78a67b32f161f5c_anonymous.png", "background_education": "S1 Teknik Geofisika", "github": "agusdwis", "description": "Hesss", "status": true, "created_at": "Sun, 21 Jun 2020 13:22:05 -0000", "update_at": null}}
[2020-06-27 09:28:34,635]{/home/alta5/Desktop/ALTA ONLINE LEARNING/BACK-END/blueprints/__init__.py:86} WARNING - REQUEST_LOG	{"method": "OPTIONS", "code": "200 OK", "uri": "/mentee/1?", "request": null, "response": {}}
[2020-06-27 09:28:34,668]{/home/alta5/Desktop/ALTA ONLINE LEARNING/BACK-END/blueprints/__init__.py:86} WARNING - REQUEST_LOG	{"method": "PATCH", "code": "200 OK", "uri": "/mentee/1?", "request": null, "response": {"id": 1, "username": "agusdwis", "full_name": "Agus D Sasongko", "email": "agusdwi@alterra.com", "address": "Solo, Indonesia", "phone": "085635467564", "place_birth": "Karanganyar", "date_birth": "17 Agustus 1995", "avatar": "https://alterra-online-learning.s3-ap-southeast-1.amazonaws.com/avatar/b9010c9fb7ce481eb78a67b32f161f5c_anonymous.png", "background_education": "S1 Teknik Geofisika", "github": "agusdwis", "description": "Hesss", "status": true, "created_at": "Sun, 21 Jun 2020 13:22:05 -0000", "update_at": null}}
[2020-06-27 09:43:29,688]{/home/alta5/Desktop/ALTA ONLINE LEARNING/BACK-END/venv/lib/python3.6/site-packages/flask/app.py:1892} ERROR - Exception on /admin [POST]
Traceback (most recent call last):
  File "/home/alta5/Desktop/ALTA ONLINE LEARNING/BACK-END/venv/lib/python3.6/site-packages/flask/app.py", line 1950, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/alta5/Desktop/ALTA ONLINE LEARNING/BACK-END/venv/lib/python3.6/site-packages/flask/app.py", line 1936, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/alta5/Desktop/ALTA ONLINE LEARNING/BACK-END/venv/lib/python3.6/site-packages/flask_restful/__init__.py", line 468, in wrapper
    resp = resource(*args, **kwargs)
  File "/home/alta5/Desktop/ALTA ONLINE LEARNING/BACK-END/venv/lib/python3.6/site-packages/flask/views.py", line 89, in view
    return self.dispatch_request(*args, **kwargs)
  File "/home/alta5/Desktop/ALTA ONLINE LEARNING/BACK-END/venv/lib/python3.6/site-packages/flask_restful/__init__.py", line 583, in dispatch_request
    resp = meth(*args, **kwargs)
  File "/home/alta5/Desktop/ALTA ONLINE LEARNING/BACK-END/blueprints/__init__.py", line 44, in wrapper
    verify_jwt_in_request()
  File "/home/alta5/Desktop/ALTA ONLINE LEARNING/BACK-END/venv/lib/python3.6/site-packages/flask_jwt_extended/view_decorators.py", line 32, in verify_jwt_in_request
    jwt_data, jwt_header = _decode_jwt_from_request(request_type='access')
  File "/home/alta5/Desktop/ALTA ONLINE LEARNING/BACK-END/venv/lib/python3.6/site-packages/flask_jwt_extended/view_decorators.py", line 294, in _decode_jwt_from_request
    decoded_token = decode_token(encoded_token, csrf_token)
  File "/home/alta5/Desktop/ALTA ONLINE LEARNING/BACK-END/venv/lib/python3.6/site-packages/flask_jwt_extended/utils.py", line 118, in decode_token
    allow_expired=allow_expired
  File "/home/alta5/Desktop/ALTA ONLINE LEARNING/BACK-END/venv/lib/python3.6/site-packages/flask_jwt_extended/tokens.py", line 140, in decode_jwt
    leeway=leeway, options=options, issuer=issuer)
  File "/home/alta5/Desktop/ALTA ONLINE LEARNING/BACK-END/venv/lib/python3.6/site-packages/jwt/api_jwt.py", line 104, in decode
    self._validate_claims(payload, merged_options, **kwargs)
  File "/home/alta5/Desktop/ALTA ONLINE LEARNING/BACK-END/venv/lib/python3.6/site-packages/jwt/api_jwt.py", line 134, in _validate_claims
    self._validate_exp(payload, now, leeway)
  File "/home/alta5/Desktop/ALTA ONLINE LEARNING/BACK-END/venv/lib/python3.6/site-packages/jwt/api_jwt.py", line 175, in _validate_exp
    raise ExpiredSignatureError('Signature has expired')
jwt.exceptions.ExpiredSignatureError: Signature has expired
[2020-06-27 09:43:29,694]{/home/alta5/Desktop/ALTA ONLINE LEARNING/BACK-END/blueprints/__init__.py:99} ERROR - REQUEST_LOG	{"method": "POST", "code": "500 INTERNAL SERVER ERROR", "uri": "/admin?", "request": null, "response": {"message": "Internal Server Error"}}
[2020-06-27 09:44:22,599]{/home/alta5/Desktop/ALTA ONLINE LEARNING/BACK-END/venv/lib/python3.6/site-packages/flask/app.py:1892} ERROR - Exception on /admin [POST]
Traceback (most recent call last):
  File "/home/alta5/Desktop/ALTA ONLINE LEARNING/BACK-END/venv/lib/python3.6/site-packages/flask/app.py", line 1950, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/alta5/Desktop/ALTA ONLINE LEARNING/BACK-END/venv/lib/python3.6/site-packages/flask/app.py", line 1936, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/alta5/Desktop/ALTA ONLINE LEARNING/BACK-END/venv/lib/python3.6/site-packages/flask_restful/__init__.py", line 468, in wrapper
    resp = resource(*args, **kwargs)
  File "/home/alta5/Desktop/ALTA ONLINE LEARNING/BACK-END/venv/lib/python3.6/site-packages/flask/views.py", line 89, in view
    return self.dispatch_request(*args, **kwargs)
  File "/home/alta5/Desktop/ALTA ONLINE LEARNING/BACK-END/venv/lib/python3.6/site-packages/flask_restful/__init__.py", line 583, in dispatch_request
    resp = meth(*args, **kwargs)
  File "/home/alta5/Desktop/ALTA ONLINE LEARNING/BACK-END/blueprints/__init__.py", line 44, in wrapper
    verify_jwt_in_request()
  File "/home/alta5/Desktop/ALTA ONLINE LEARNING/BACK-END/venv/lib/python3.6/site-packages/flask_jwt_extended/view_decorators.py", line 32, in verify_jwt_in_request
    jwt_data, jwt_header = _decode_jwt_from_request(request_type='access')
  File "/home/alta5/Desktop/ALTA ONLINE LEARNING/BACK-END/venv/lib/python3.6/site-packages/flask_jwt_extended/view_decorators.py", line 314, in _decode_jwt_from_request
    raise NoAuthorizationError(errors[0])
flask_jwt_extended.exceptions.NoAuthorizationError: Missing Authorization Header
[2020-06-27 09:44:22,600]{/home/alta5/Desktop/ALTA ONLINE LEARNING/BACK-END/blueprints/__init__.py:99} ERROR - REQUEST_LOG	{"method": "POST", "code": "500 INTERNAL SERVER ERROR", "uri": "/admin?", "request": null, "response": {"message": "Internal Server Error"}}
[2020-06-27 09:45:52,230]{/home/alta5/Desktop/ALTA ONLINE LEARNING/BACK-END/venv/lib/python3.6/site-packages/flask/app.py:1892} ERROR - Exception on /admin [POST]
Traceback (most recent call last):
  File "/home/alta5/Desktop/ALTA ONLINE LEARNING/BACK-END/venv/lib/python3.6/site-packages/flask/app.py", line 1950, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/alta5/Desktop/ALTA ONLINE LEARNING/BACK-END/venv/lib/python3.6/site-packages/flask/app.py", line 1936, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/alta5/Desktop/ALTA ONLINE LEARNING/BACK-END/venv/lib/python3.6/site-packages/flask_restful/__init__.py", line 468, in wrapper
    resp = resource(*args, **kwargs)
  File "/home/alta5/Desktop/ALTA ONLINE LEARNING/BACK-END/venv/lib/python3.6/site-packages/flask/views.py", line 89, in view
    return self.dispatch_request(*args, **kwargs)
  File "/home/alta5/Desktop/ALTA ONLINE LEARNING/BACK-END/venv/lib/python3.6/site-packages/flask_restful/__init__.py", line 583, in dispatch_request
    resp = meth(*args, **kwargs)
  File "/home/alta5/Desktop/ALTA ONLINE LEARNING/BACK-END/blueprints/__init__.py", line 44, in wrapper
    verify_jwt_in_request()
  File "/home/alta5/Desktop/ALTA ONLINE LEARNING/BACK-END/venv/lib/python3.6/site-packages/flask_jwt_extended/view_decorators.py", line 32, in verify_jwt_in_request
    jwt_data, jwt_header = _decode_jwt_from_request(request_type='access')
  File "/home/alta5/Desktop/ALTA ONLINE LEARNING/BACK-END/venv/lib/python3.6/site-packages/flask_jwt_extended/view_decorators.py", line 314, in _decode_jwt_from_request
    raise NoAuthorizationError(errors[0])
flask_jwt_extended.exceptions.NoAuthorizationError: Missing Authorization Header
[2020-06-27 09:45:52,233]{/home/alta5/Desktop/ALTA ONLINE LEARNING/BACK-END/blueprints/__init__.py:99} ERROR - REQUEST_LOG	{"method": "POST", "code": "500 INTERNAL SERVER ERROR", "uri": "/admin?", "request": null, "response": {"message": "Internal Server Error"}}
